node_helper.js

```diff
@@ -9,8 +9,8 @@ module.exports = NodeHelper.create({
 
       axios.get(url).then(({ data }) => {
         const $ = cheerio.load(data);
-  
-        const container = $(".entry--1qTEOKKt")[0];
+
+        const container = $("._1qTEOKKt")[0];
```

Comment:
May I suggest using a "begins  with"  class selector?
```suggestion
        const container = $("div[class^='entry--']")[0];
```